source/importing-css-assets-and-javascript/index.html.md.erb

```diff
@@ -176,6 +176,13 @@ You can select and initialise a specific component by using its `data-module` at
   </script>
 ```
 
+When initialised individually, components will throw errors if GOV.UK Frontend is not supported, some key markup is missing or their configuration is invalid. 
```

Comment:
Would you mind if we moved this whole section up where `initAll()` is the default?

This current section is optional and might be missed, but we want people to notice the good stuff 🙌

For most of our users on `initAll()` we could list some examples logged to the console:

>Errors from components will be logged in the browser's console.
>
>For example, when:
>
>* GOV.UK Frontend is not supported in the current browser
>* Component templates have missing changes from our release notes
>* Component JavaScript configuration does not match our documentation

Which means we can now keep this section simple:

```suggestion
When initialised individually, errors are thrown rather than logged and must be handled with a [`try/catch` block](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch).
```

Reply:
Do you mind if we don't give developer users the option to let their service crash?